### ISSUE TITLE
Improve accessibliity of generated output

### ISSUE THREAD

**Issue by simoncoggins** [anvilco/spectaql#396]
We are integrating spectaql within our product and while testing ran an accessibility review. It came back with a few small issues and one of our developers has written a pull request to fix them so creating an issue to track it.

**Comment by simoncoggins** [comment]
Thanks to @jackrhumphrey for the PR

**Comment by newhouse** [comment]
Sorry I lost track of this...can you elaborate on what the idea is here?

**Comment by jackrhumphrey** [comment]
Hi we found some mild accessibility failures while testing our spectaql implementation.

Our automated testing detected a heading rank being skipped by going from h3 to h5 https://www.w3.org/WAI/tutorials/page-structure/headings/

However, this was really because the h4 element was hidden using display: none and couldn't be detected by the test software.

Accessible hiding can be better achieved by making use of a visually hidden class: https://a11y-guidelines.orange.com/en/web/components-examples/accessible-hiding/ This keeps them available to screen readers and keeps the page structure in tact.

The "All Topics" hamburger menu is not accessible: this could be fixed by adding an aria-label to the button, but we can make use of the visually hidden class here as well--this has some modest advantages to it as described in the article.

We also changed the text to "All topics" (sentence case)--not an accessibility failure but a useful convention to follow https://practicaltypography.com/headings.html

The aria-label for the "Close menu" button was correct, but again could make use of the visually hidden class instead.

**Comment by newhouse** [comment]
cc @benogle
